changes.go

```diff
@@ -367,6 +367,15 @@ type AttentionSetInfo struct {
 // the edits to span newlines.
 type DiffIntralineInfo [][2]int
 
+// ChangeInput entity contains information about creating a new change.
+type ChangeInput struct {
+	Project string `json:"project"`
+	Branch  string `json:"branch"`
+	Subject string `json:"subject"`
+	Topic   string `json:"topic,omitempty"`
+	Status  string `json:"status,omitempty"`
```

Comment:
What do you think about to add the other properties listed in https://gerrit-review.googlesource.com/Documentation/rest-api-changes.html#change-input as well?

Reply:
See #97